packages/ui/src/styles/base/settings.styl

```diff
@@ -2,21 +2,28 @@
   // fonts
   --uie-font-family-fallback 'Helvetica', Arial, Tahoma, Verdana, sans-serif
   --uie-font-family-code 'Fira Code', Hasklig, 'Source Code Pro', 'DejaVu Sans Mono', Hack, Meslo, Menlo, Inconsolata, Consolas, 'Liberation Mono', Courier, monospace
-  --uie-font-family-bold 'UIE Merriweathersans bold', var(--uie-font-family-fallback)
-  --uie-font-family-light 'UIE Merriweathersans light', var(--uie-font-family-fallback)
-  --uie-font-family-regular 'UIE Merriweathersans regular', var(--uie-font-family-fallback)
-  --uie-font-family-bolditalic 'UIE Merriweathersans bolditalic', var(--uie-font-family-fallback)
-  --uie-font-family-lightitalic 'UIE Merriweathersans lightitalic', var(--uie-font-family-fallback)
-
-  --uie-font-weight-bold normal
-  --uie-font-weight-regular normal
-  --uie-font-weight-light normal
-  --uie-font-weight-italic normal
-  --uie-font-style-italic normal
-
-  --uie-base-font-size-s 14px
-  --uie-base-font-size-m 15px
-  --uie-base-font-size-l 16px
+  --uie-font-family-bold 'Inter', var(--uie-font-family-fallback)
+  --uie-font-family-light 'Inter', var(--uie-font-family-fallback)
+  --uie-font-family-regular 'Inter', var(--uie-font-family-fallback)
+  --uie-font-family-bolditalic 'Inter', var(--uie-font-family-fallback)
+  --uie-font-family-lightitalic 'Inter', var(--uie-font-family-fallback)
+
+  --uie-font-weight-thin 100
+  --uie-font-weight-extralight 200
+  --uie-font-weight-light 300
+  --uie-font-weight-regular 400
+  --uie-font-weight-italic 400
+  --uie-font-weight-medium 500
+  --uie-font-weight-semibold 600
+  --uie-font-weight-bold 700
+  --uie-font-weight-extrabold 800
+  --uie-font-weight-black 900
+  --uie-font-style-italic italic
+
+  --uie-base-font-size 16px
+  --uie-base-font-size-s var(--uie-base-font-size)
+  --uie-base-font-size-m var(--uie-base-font-size)
+  --uie-base-font-size-l var(--uie-base-font-size)
```

Comment:
This looks weird – what's the reasoning behind this? Maybe we should just refactor it to get rid of the different sizes then.

Reply:
When I first tested Inter font on mobile it looked too small in 14px (the original value of `--uie-base-font-size-s`) and thought it might be worth rationalising the font size throughout the viewports which would also bring in more consistency. Plus there's an "issue" (is actually an accessibility feature according to Apple) on iOS or iPadOS where input fields (like the search bar in the topbar) with font-size lower than 16px will zoom in the page when focus is set on them, which I always find a bit annoying.

I thought it might be better to refactor it but then I considered that we should keep the original base font variable in place for backwards compatibility with existing projects where developers may overwritten those variables and made use of them in other places. Removing them could "break" those projects. Unless you think different...

We can also not introduce the new `--uie-base-font-size` variable and set the original ones to all have 16px by default.

Reply:
You are right about the backwards compatibility – good point. Let‘s keep it then 😀